Edit projects page

MizuBishi opened the pull request.

MizuBishi committed:
Added first draft of EditProjectPage.

MizuBishi committed:
Deleted component ProjectList. ProjectList is only needed in EditProjectPage. Therefore is the list directly implemented in EditProjectPage.

joelmeiller committed:
Fix: Extended EditTeamPage story with store

joelmeiller commented on src/ui/pages/EditProjectPage.jsx:
```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 
 import {Table, TableBody, TableHeader, TableHeaderColumn, TableRow, TableRowColumn} from 'material-ui/Table';
 import {cyan500} from 'material-ui/styles/colors';
```
Sollten wir durch css variable ersetzen.

joelmeiller commented:
Eigentlich war die Idee eine neue Page zu erstellen, auf der der Benutzer ein Projekt bearbeiten kann. Das fehlt jetzt noch. Aber wir können das nachher zusammen anschauen ;) Ich warte mal noch mit mergen

MizuBishi commented:
Hoi Joel 
Stimmt, das war wohl ein kleines Missverständnis, sorry! Ich mache das jetzt noch kurz.

MizuBishi committed:
Add ProjectOverview in containers but not finished yet.

MizuBishi committed:
Edit existing EditProjectPage to ProjectView and add ned EditProjectPage.
The following Cases are open:
//declaration PropTypes
//add component "EditableProject.js in components?"
//Make props in Header for Project. Concatendate of Strings.

MizuBishi commented:
Ich habe eine neue Page hinzugefügt. Ist aber noch nicht fertig. Tasks:
//open:
//declaration PropTypes
//add component "EditableProject.js in components?"
//Make props in Header for Project. Concatendate of Strings.
